### PR TITLE
Upgrade yansi: 0.5.1 -> 1.0.1

### PR DESCRIPTION
**Pull request by nc7s**
Release note: https://github.com/SergioBenitez/yansi/releases/tag/v1.0.0

It'd be nice to have a version bump for this (patch level suffices), so I can straight update it in Debian rather than having to carry a patch ;)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -249,20 +249,13 @@ fn main() {
     let args = Cli::parse();
 
     // Determine the usage of styles
-    #[cfg(target_os = "windows")]
-    let ansi_support = yansi::Paint::enable_windows_ascii();
```
Comment (niklasmohrin) [inline src/main.rs:253]: I suppose by https://docs.rs/yansi/1.0.1/yansi/index.html#windows we can omit this. Fair enough

**Review by niklasmohrin — APPROVED** [review]

**Comment by niklasmohrin** [comment]
Cool! Do you know of any other dependencies updates we can do that would declutter the package index only requiring a patch-version bump?

File: src/main.rs (hunk 1 above)
Comment (nc7s) [reply src/main.rs:253]: Yeah the commit message explained ;)

**Comment by nc7s** [comment]
Not really, as least in Debian. Thanks for asking ;)

File: src/main.rs (hunk 1 above)
Comment (niklasmohrin) [reply src/main.rs:253]: Ah, I didn't see that, I'll make sure to include it in the merged commit

File: src/main.rs [hunk 2]
```diff
@@ -249,20 +249,16 @@ fn main() {
...
-    #[cfg(not(target_os = "windows"))]
-    let ansi_support = true;
     let enable_styles = match args.color.unwrap_or_default() {
         // Attempt to use styling if instructed
-        ColorOptions::Always => true,
+        ColorOptions::Always => {
+            yansi::enable(); // disable yansi's automatic detection for ANSI support on Windows
```
Comment (niklasmohrin) [inline src/main.rs:255]: @nc7s Sorry to bother you again, but how do you feel about this change I added? It should now be exactly the previous behavior right?

Comment (nc7s) [reply src/main.rs:255]: I don't really know enough to judge this, from a glance LGTM?

**Pull request merged**